Makefile

```diff
@@ -113,16 +126,7 @@ update-third-party:
 .PHONY: refresh-sample-testdata out/bincapz
 refresh-sample-testdata: clone-samples out/bincapz
 	cp ./test_data/refresh-testdata.sh samples/
```

Comment:
note: in a future PR, we can switch to running this script from the `test_data` directory.